src/test/java/com/vinylteam/vinyl/service/impl/DefaultOneVinylOffersServiceTest.java

```diff
@@ -57,15 +59,15 @@ void givenVinylListWithNoOffer_whenHasOffersSetToFalse_thenCorrect() {
         var vinylsList = dataGenerator
                 .getUniqueVinylsList()
                 .stream()
-                .filter(UniqueVinyl::hasOffers)
+                .filter(UniqueVinyl::isHasOffers)
```

Comment:
Either you forgot to commit something or it does not compile
There is no such method in UniqueVinyl - isHasOffers, only hasOffers